com.gitlab.jakobdev.jdnbtexplorer.metainfo.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this file should be submitted upstream.

also it should be installed in the manifest.

Reply:
it no being installed is why build failed. Change above is required too.